lib/cc/engine/bundler_audit/result.rb

```diff
@@ -0,0 +1,85 @@
+module CC
+  module Engine
+    module BundlerAudit
+      class Result
```

Comment:
I'm personally okay with the higher level test including an actual `Gemfile.lock` file on Analyzer here.